components/purchase_modal/purchase.scss

```diff
@@ -298,24 +149,156 @@
                 }
             }
 
-            .payment-note {
-                margin-top: 5px;
-                margin-bottom: 10px;
-                color: rgba(var(--sys-center-channel-color-rgb), 0.72);
+            .monthly-text {
+                align-self: center;
+                margin-left: 5px;
                 font-size: 14px;
-                font-weight: 400;
-                line-height: 20px;
+                font-weight: normal;
+            }
+
+            .plan_comparison {
+                margin-bottom: 51px;
+                text-align: center;
 
+                button,
                 a {
-                    color: var(--button-bg);
+                    width: 100%;
+                    height: 40px;
+                    padding: 10px 24px;
+                    border: none;
+                    background: none;
+                    color: #1c58d9;
+                    font-size: 14px;
+                    font-weight: 600;
+                    line-height: 20px;
                 }
             }
 
-            .monthly-text {
-                align-self: center;
-                margin-left: 5px;
-                font-size: 14px;
-                font-weight: normal;
+            .PlanCard {
+                position: relative;
+                width: 280px;
+                height: 428px;
+
+                .planLabel {
+                    position: absolute;
+                    top: -44px;
+                    left: 52px;
+                    display: flex;
+                    width: 184px;
+                    align-items: center;
+                    justify-content: center;
+                    padding: 14px;
+                    border-radius: 13px 13px 0 0;
+                    font-family: 'Open Sans';
+                    font-size: 12px;
+                    font-style: normal;
+                    font-weight: 600;
+                    gap: 5px;
+                    line-height: 16px;
+                }
+
+                .top {
+                    height: 18px;
+                    border-radius: 4px 4px 0 0;
+                }
+
+                .bottom {
+                    height: 363px;
+                    padding-right: 24px;
+                    padding-left: 24px;
+                    border: 1px solid #dadfe5;
+                    background-color: var(--center-channel-bg);
+                    border-radius: 0 0 4px 4px;
+
+                    .plan_action_btn {
+                        width: 100%;
+                        height: 40px;
+                        padding: 10px 24px;
+                        border: none;
+                        background: none;
+                        border-radius: 4px;
+                        color: #1c58d9;
+                        font-size: 14px;
+                        font-weight: 700;
+                        line-height: 14px;
+
+                        &.grayed {
+                            background: rgba(63, 67, 80, 0.08);
```

Comment:
For the three `63,67,80` usages, can we use `rgba(var(--center-channel-text-rgb), <alpha value>)`